readthedocs/api/v3/serializers.py

```diff
@@ -663,6 +693,12 @@ def __init__(self, *args, **kwargs):
         if settings.RTD_ALLOW_ORGANIZATIONS:
             self.fields.pop("users", None)
 
+        # If privacy levels are not allowed,
+        # everything is public, we don't allow changing it.
+        if not settings.ALLOW_PRIVATE_REPOS:
+            self.fields.pop("privacy_level")
+            self.fields.pop("external_builds_privacy_level")
```

Comment:
It maybe makes sense to have a mixin (e.g. `PrivacyLevelMixin`) for this, so we don't have to duplicate the logic on these serializers.

Reply:
Hrm... Thinking a little more about this. Why we don't expose the same response instead? In commercial it could be `private`/`public` but in community it will always be `public`.

That will simplify things a lot and remove confusions. Besides, we will have consistent responses in both platforms.